cmd/odf/restore/crds.go

```diff
@@ -1,12 +1,54 @@
 package restore
 
 import (
+	"fmt"
+	"strings"
+
 	"github.com/red-hat-storage/odf-cli/cmd/odf/root"
 	"github.com/rook/kubectl-rook-ceph/pkg/k8sutil"
 	pkgrestore "github.com/rook/kubectl-rook-ceph/pkg/restore"
 	"github.com/spf13/cobra"
 )
 
+// groupVersions defines the supported CRD groups and their corresponding API versions.
+var groupVersions = map[string]string{
+	"ocs.openshift.io":       "v1",
+	"ceph.rook.io":           "v1",
+	"storage.k8s.io":         "v1",
+	"csi.ceph.io":            "v1",
+	"odf.openshift.io":       "v1alpha1",
+	"noobaa.io":              "v1alpha1",
+	"csiaddons.openshift.io": "v1alpha1",
+}
+
+// keys returns the keys of a string map. It is used to print out supported group names.
+func keys(m map[string]string) []string {
+	out := make([]string, 0, len(m))
+	for k := range m {
+		out = append(out, k)
+	}
+	return out
+}
+
+// parseFullyQualifiedCRD takes a fully qualified CRD type of the form "resource.group"
+// (for example, "cephclusters.ceph.rook.io") and returns the resource name, group name, and
+// the API version associated with that group. It returns an error if the format is invalid
+// or the group is not recognized.
+func parseFullyQualifiedCRD(fqcrd string) (resourceName, groupName, version string, err error) {
```

Comment:
Please add a unit test for this method. Unit tests are always valuable for parsing.